lib/zcaps.js

```diff
@@ -14,12 +15,13 @@ const SUPPORTED_KEY_TYPES = [
   'Sha256HmacKey2019',
   'X25519KeyAgreementKey2019'
 ];
+const {config} = bedrock;
```

Comment:
recommend just destructing config on L9 as it does not appear `bedrock` is being used otherwise.